config/locales/en.yml

```diff
@@ -196,6 +196,8 @@ en:
       long: ! '%B %d, %Y %H:%M'
       short: ! '%d %b %H:%M'
     pm: pm
+  placeholder:
+    full_name_or_nickname_or_event_name: "Full Name or Nickname or Event Name"
```

Comment:
Please, take a look to the structure of `config/locales`. `en.yml` is used for messages coming from the framework (Rails), in the same way that `device.en.yml` is used for messages coming from the Devise gem and so on.

Thus, the right place for this string would be `application.en.yml'.

On the other hand, the name of the symbol doesn't have to match the string 1:1. That's one of the main points of using labels instead of the gettext approach. Please use a meaningful but short name.